### ISSUE TITLE
[5.0.0RC1] bad command in config.m4

### ISSUE THREAD

**Issue by remicollet** [phpredis/phpredis#1577]
```
checking for redis msgpack support... expr: syntax error: unexpected argument '2.0.3'
enabled

```

**Comment by remicollet** [comment]
IMHO this version check have no real value...

BTW, probably a check for JSON extension is needed, as some may build without it.

**Comment by yatsukhnenko** [comment]
Thanks @remicollet 
Could you give me an advice how to correctly check version of external lib?

**Comment by yatsukhnenko** [comment]
> probably a check for JSON extension is needed

OK, I'll add

**Comment by remicollet** [comment]
> Could you give me an advice how to correctly check version of external lib?

Sorry, no idea for a clean way

**Comment by yatsukhnenko** [comment]
I think I'll just remove that checking. It is not business of phpredis which version of libs installed in the system :)

**Comment by michael-grunder** [comment]
Hi @yatsukhnenko and @remicollet 

What do you guys think about a simple grep solution like f0d3231?  It's not particularly robust but it does appear to work on my machine.  I worry a lot of users will be confused if it passes configure and then fails to compile.

Open to other suggestions of course.

**Comment by yatsukhnenko** [comment]
@michael-grunder there is a note in INSTALL.markdown about minimum required version

**Comment by yatsukhnenko** [comment]
@michael-grunder I've added checking msgpack version for testing PHP5 and PHP7 builds on travis. Now this check is unnecessary.

**Comment by yatsukhnenko** [comment]
@remicollet could you test the latest changes from `develop` branch?

**Comment by remicollet** [comment]
LGTM

**Comment by yatsukhnenko** [comment]
Fixed in 5.0.0RC2
